dist.ini

```diff
@@ -0,0 +1,68 @@
+name    = Pod-Usage
+author  = Marek Rouchal <marekr@cpan.org>
+license = Perl_5
+copyright_holder = Marek Rouchal <marekr@cpan.org>
+copyright_year   = 2013
+
+[ReadmeAnyFromPod / Markdown_Readme]
+type = gfm
+source_filename = lib/Pod/Usage.pm
+filename = README.md
+location = root
+
+[Regenerate::AfterReleasers]
+plugin = Markdown_Readme
+
+[@Starter::Git]
+revision = 4
+managed_versions = 1
+installer = MakeMaker::Awesome
+-remove = MakeMaker::Awesome
+RewriteVersion.global = 1
+NextRelease.format = %-9v %{yyyy-MM-dd}d
+regenerate = Makefile.PL
+regenerate = META.json
+regenerate = README.md
+regenerate = LICENSE
+regenerate = t/00-report-prereqs.t
+Git::Check.allow_dirty = META.json
+ExecDir.dir = script
+
+;MakeMaker::Awesome.exe_file = scripts/pod2usage
+;MakeMaker::Awesome.WriteMakefile_arg[] = PL_FILES => { 'scripts/pod2usage.PL' => 'scripts/pod2usage' }
+
+; use a custom DistMaker to adjust EXE_FILES on VMS
+[=inc::MyDistMakeMaker / MyDistMakeMaker]
```

Comment:
This is not necessary, you can use a `$^O` condition within WriteMakefile_args (https://metacpan.org/pod/Dist::Zilla::Plugin::MakeMaker::Awesome#WriteMakefile_arg even has an example)

Reply:
I've adjusted it, unfortunately it seems that header/footer cannot be use multiple times from `@Starter::Git` so I worked around it with c2b98843

Reply:
Take a look at https://metacpan.org/pod/Config::MVP::Slicer#CONFIGURATION-SYNTAX for how multi-value options need to be configured when slicing into the bundle, I can explain more on IRC if needed. WriteMakefile_arg and footer are both multi-value options.

Reply:
thanks, this is now fixed by c6fa05718feb43ae8da15d804b1a19523adbe24b